src/pages/ProposalList.tsx

```diff
@@ -64,4 +63,4 @@ const ProposalListLoader = () => {
 };
 
 const ProposalList = withOidcSecure(ProposalListLoader);
-export { ProposalList };
+export { ProposalList, mapFieldNames };
```

Comment:
Rather than export `mapFieldNames` from here, if we're going to re-use it, let's move it to a utilities file — perhaps `/src/utils/api.tsx` or `/src/utils/baseFields.tsx`? — and then re-import it wherever needed.

Reply:
I just realized that we have a competing pattern of putting some one-off utility files directly at the root of `/src/`, e.g. [`/src/map-proposals.ts`](https://github.com/PhilanthropyDataCommons/front-end/blob/main/src/map-proposals.ts). tbh I think that bears cleaning up — but I'll open a separate task for that. Go ahead and put any new utility files in `/src/utils/`!